packages/p2p/src/components/modal-manager/modals/business-hour-modal/business-hour-modal-edit/business-hour-modal-edit.tsx

```diff
@@ -140,6 +265,8 @@ const BusinessHourModalEdit = ({ data }: TBusinessHourModalEditProps) => {
             </div>
         </div>
     );
-};
+});
+
+BusinessHourModalEdit.displayName = 'BusinessHourModalEdit';
```

Comment:
is this needed?

Reply:
yes, since its using a wrapper and it throws an error if the display name is not given